### ISSUE TITLE
[med] JAWS+Chrome: Create New Group: Incorrect focus upon selecting the contact

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5670]
## Action Performed:
Prerequisites:
- User has created personal account and is logged into same.
- User has several contacts or recent chats that display in the new group panel.

1. Using JAWS+Chrome, open https://staging.new.expensify.com/
2. Tab over to the new chat floating action icon button ( + ) in the left side panel and press Enter.
3. Tab to the 'New Group' option in the flyout menu and press Enter.
4. In the 'New Group' panel, Tab over to any of the contacts displayed and press Enter. Note what is focused and announced.

## Expected Result:
When a user selects or deselects a user for the group chat, the contact is either moved to the top of the list of contacts or removed from the top and returned to it's original place in the list. The focus should remain on the selected contact and the user notified that the contact has been selected and moved. Alternatively the user is notified of any custom focus behavior prior to selection.

## Actual Result:
When the user selects or deselects a contact for the group chat the contact is moved as described, but nothing is announced and the focus goes to the search field.

HTML
`<div class="css-1dbjc4n r-13awgt0" style="padding-bottom: 0px; padding-top: 0px;"><div class="css-1dbjc4n r-1777fci r-1udh08x r-13qz1uu" style="display: flex; height: 65px; padding-left: 20px;"><div class="css-1dbjc4n r-1awozwy r-16y2uox r-1udh08x" style="display: flex; flex-direction: row; justify-content: space-between; -webkit-box-orient: horizontal; -webkit-box-direction: normal; -webkit-box-pack: justify;"><div class="css-1dbjc4n r-13awgt0" style="flex-direction: row; -webkit-box-orient: horizontal; -webkit-box-direction: normal;"><div dir="auto" class="css-901oao css-cens5h" style="-webkit-line-clamp: 2; color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 17px; font-weight: 700;">New Group</div></div>...</div>`

## Workaround:
No workaround

## Area issue was found in:
Create new group overlay

## Failed WCAG checkpoints
2.4.3

## User impact:
Keyboard and screen reader users will be very confused by the moving contact button displacing their position on the page without warning and the shift from button activation to a text input field.

## Suggested resolution:
Do not move the focus to the search field when a contact is selected or deselected. Have the focus remain on the button and notify the user the contact has been added to the group.
https://www.w3.org/WAI/WCAG21/Understanding/on-input.html

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/136174055-a23c9eb7-0d60-4829-98de-8ce1fe2305c0.mp4

![Bug5186963_Chrome_Jaws-Create_New_Group-Wrong_focus_when_user_selects_contact](https://user-images.githubusercontent.com/88733897/136174069-58206d12-69ef-4e73-a0a9-dd566948ab7a.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
